ofl/muktamahee/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,18 @@
+<p>
+Mukta is a Unicode compliant, versatile, contemporary, humanist, mono-linear typeface available in seven weights, supporting Devanagari, Gujarati, Tamil and Latin scripts. 
+This type family is a libre licensed version of Ek's self-titled multi-script project, an ongoing effort to develop a unified type family for each Indian script. 
+The goal is to build one harmonious family across all Indian scripts without letting the visual features of one script dominate over others. 
+This ensures that the fonts can be used successfully for both single and multi-script purposes. 
+</p>
+<p>
+Mukta was designed by Girish Dalvi and Yashodeep Gholap. 
```

Comment:
<a href="https://fonts.google.com/specimen/Mukta"> Mukta </a> was designed by Girish Dalvi and Yashodeep Gholap.